crates/weaver_resolved_schema/src/attribute.rs

```diff
@@ -91,7 +91,7 @@ pub struct UnresolvedAttribute {
 
 /// An internal reference to an attribute in the catalog.
 #[derive(
-    Serialize, Deserialize, Debug, Copy, Clone, Eq, PartialEq, PartialOrd, Ord, JsonSchema,
+    Serialize, Deserialize, Debug, Copy, Clone, Eq, PartialEq, Hash, PartialOrd, Ord, JsonSchema,
```

Comment:
At the time of adding this was required as the compilation failed because this was not tagged as implementing this, this is (probably) a left over of the now removed definition to return an Attribute Group with Ref. Will validate and remove if no longer required.

Reply:
Removed as no longer required